src/main/java/com/mindee/product/financialdocument/FinancialDocumentV1LineItem.java

```diff
@@ -101,11 +130,7 @@ public String toString() {
   private Map<String, String> printableValues() {
     Map<String, String> printable = new HashMap<>();
 
-    String descriptionSummary = (this.description != null ? this.description : "");
-    if (descriptionSummary.length() >= 36) {
-      descriptionSummary = descriptionSummary.substring(0, 33) + "...";
-    }
-    printable.put("description", descriptionSummary);
+    printable.put("description", SummaryHelper.formatForDisplay(this.description, null));
     printable.put("productCode", SummaryHelper.formatForDisplay(this.productCode, null));
```

Comment:
Given that the return from `tablePrintableValues()` is already run through `formatForDisplay()`, there is no need to run it again here.

Reply:
My bad, was error with GH format in the PR.